nservicebus/testing/index.md

```diff
@@ -66,4 +66,6 @@ snippet:TestingAdditionalDependencies
 
 ## Constructor injected bus
 
+NOTE: The `IBus` interface was deprecated in Version 6, and replaced with the contextual `IMessageHandlerContext` parameter on the `IHandleMessages<T>.Handle()` methods.
```

Comment:
@indualagarsamy when previously adding deprecation notice, I used to link to the upgrade guide, but we realized this wasn't a good idea. Now, the deprecation notice suggests what to do instead, e.g. http://docs.particular.net/nservicebus/containers/injecting-ibus

We don't (currently) have a dedicated doc for handler context methods. Maybe we should add it? @timbussmann